charts/lightdash/templates/scheduler.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "lightdash.fullname" . }}-scheduler
+  labels:
+    {{- include "lightdash.labels" . | nindent 4 }}
+spec:
+  {{- if .Values.scheduler.enabled }}
+  replicas: {{ .Values.scheduler.replicas }}
+  {{ else }}
+  replicas: 0 
```

Comment:
We should just remove the whole deployment. It would be confusing to me as a dev if there's a deployment with zero replicas "should it have zero? am I missing something?"